cmd/e2e_test.go

```diff
@@ -1,15 +1,34 @@
 package cmd
 
 import (
+	"context"
 	"testing"
 
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/client"
 	_ "github.com/redhat-appstudio/e2e-tests/pkg/tests/common"
 	_ "github.com/redhat-appstudio/e2e-tests/pkg/tests/has"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+)
+
+const (
+	APPLICATION_SERVICE_GITHUB_TOKEN_SECRET = "has-github-token"
+	APPLICATION_SERVICE_NAMESPACE           = "application-service"
 )
 
 var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
+	KubeClient, err := client.NewK8SClient()
+	gomega.Expect(err).To(gomega.BeNil(), "Error when trying to start a new K8S client")
```

Comment:
I guess it's just a matter of preference, but this seems to be more in line with what gomega suggests: https://onsi.github.io/gomega/#asserting-on-errors
Same for the next assertion.
Not a big deal though ;-)
```suggestion
	gomega.Expect(err).ShouldNot(gomega.HaveOccurred(), "Error when trying to start a new K8S client")
```

Reply:
Hi @rhopp! Thank you for the tips. I've fixed both `gomega` statements in a new commit, [7f81167](https://github.com/redhat-appstudio/e2e-tests/pull/17/commits/7f81167acf9af9cb5658cdd58641272b88be6e32)